internal/apiserver/draft.go

```diff
@@ -60,6 +78,17 @@ func (a *App) getMyDraft(res http.ResponseWriter, req *http.Request) error {
 }
 
 func (a *App) createMyDraft(res http.ResponseWriter, req *http.Request) error {
+	ctx := req.Context()
+
+	// Limit number of drafts
+	drafts, err := a.cache.GetDrafts(ctx)
```

Comment:
maybe we can extract getting drafts from GET /drafts to a function and use it here, so we can delete old drafts and potentially make space for this new one